### ISSUE TITLE
Char encoding error

### ISSUE THREAD

**Issue by chabapok** [akuleshov7/ktoml#260]
If we try encode Char, it encoded with double quotes. If we try to decode, it fails. Decoder wants single-quoted chars:

```
@Serializable
data class Foo(val foo: Char= 'x')

fun main(){
    val s = Toml().encodeToString(Foo('f'))
    println(s) // foo = "f"

    // decode exception
    val config = Toml().decodeFromString<Foo>(s)

    // single-quoted decoded ok
    Toml().decodeFromString<Foo>("doo='f'")
}
```

**Comment by akuleshov7** [comment]
Thanks for this report! 

@chabapok, according to TOML spec there is no such `Char` type, so this is just or own idea in TOML to look more Kotlin-like and decode `Char` as literal string with single quote.

Probably we need to make it more convenient and do encoding and decoding similar.

Will fix it, thanks!

**Comment by akuleshov7** [comment]
fixed in #261
